training/common/usr/libexec/upgrade-informer

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Run the command and capture its output
+output=$(bootc upgrade --check | sed -e 1q)
+message_file="/etc/motd.d/upgrade-message"
+bootc_auth="/etc/ostree/auth.json"
```

Comment:
how will this file get there? what happens if it doesnt exist?

Reply:
Currently we talked that user will need to add it.
We can run command like :
`podman login registry.redhat.io --username user --password pass --verbose | grep Used | awk '{print $2}' | xargs -I{} cp {} /etc/ostree/auth.json
`
There is a talk in bootc to add some common auth file between bootc and podman but it is still talks

Reply:
upgrade-informer would run before the user has a chance to authenticate unless you're collecting it in firstboot (which we talked about doing?)

Reply:
We can create doc how to add creds on boot but i am not sure how we can make customers to do it.
I can fail nice in case we can't reach registry and exit 0

Reply:
We can add systemd service that can collect username/password as env variables and connect system to registry + insights on boot? Will it be more helpful? 
I can block this service if /etc/ostree/auth.json was not set? Not sure what is better, not to run or try and fail gracefully in case no auth needed.

Reply:
What will happened with current code 
`Jul 15 13:23:50 localhost upgrade-informer[1645]: ERROR Upgrading: Creating importer: Failed to invoke skopeo proxy method OpenImage: remote error: reading manifest v3.15 in quay.io/itsoiref/nvidia-bootc-private: unauthorized: access to the requested re>
Jul 15 13:23:50 localhost upgrade-informer[1643]: No upgrade was found
`
It will not fail the service and just log the error
@jeremyeder i think it should be safe enough for now, WDYT?